swarm/runtime/types/macro_types.py

```
@@ -120,6 +120,18 @@ def record_use(self) -> None:
         """Record that this rule was used."""
         self.uses += 1
 
+    def clone(self) -> "MacroRoutingRule":
```

Ruff (0.15.11)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)